Modules/Sources/FormattableContentKit/Extensions/NSMutableAttributedString+Extensions.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+extension NSAttributedString {
+    /// This helper method returns a new NSAttributedString instance, with all of the the leading / trailing newLines
+    /// characters removed.
+    ///
+    func trimNewlines() -> NSAttributedString {
```

Comment:
I copied this from `WordPressKit` – it shouldn't use helpers from it or depend on it. Since it was only used in one place, I added it directly to `FormattableTestKit`.